setup.py

```diff
@@ -0,0 +1,61 @@
+"""
+Copyright [1999-2015] Wellcome Trust Sanger Institute and the EMBL-European Bioinformatics Institute
+Copyright [2016-2020] EMBL-European Bioinformatics Institute
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+from pathlib import Path
+
+from setuptools import setup, find_packages
+
+
+with open(os.path.join(os.path.dirname(__file__), 'README.md')) as f:
+    readme = f.read()
+
+with open(os.path.join(os.path.dirname(__file__), 'LICENSE')) as f:
+    license_ct = f.read()
+
+with open(os.path.join(os.path.dirname(__file__), 'VERSION')) as f:
+    version = f.read()
+
+
+def import_requirements():
+    """Import ``requirements.txt`` file located at the root of the repository."""
+    with open(Path(__file__).parent / 'requirements.txt') as file:
+        return [line.strip(' \n') for line in file.readlines()]
+
+
+setup(
+    name='ensembl-compara',
+    version=version,
+    description="Ensembl Compara's python library",
+    long_description=readme,
+    author='Ensembl Compara',
+    author_email='ensembl-compara@ebi.ac.uk',
```

Comment:
```suggestion
    author_email='helpdesk@ensembl.org',
```

Reply:
I will check whether we should be using dev@ or helpdesk@. But definitely not the compara one since it's private

Reply:
I think I used this because in the repo I'm using as template their email was the one for production. Anyway, changed.